src/main/scala/subsystem/HasTiles.scala

```diff
@@ -219,6 +233,7 @@ trait HasTileNotificationSinks { this: LazyModule =>
 trait DefaultTileContextType
   extends Attachable
   with HasTileInterruptSources
+  with HasTileNMISources
```

Comment:
I'm not sure that `HasTileNMISources`/`connectNMI` needed to be separate from `HasTileInterruptSources`/`connectInterrupts` respectively, which I say because a) they are notionally interrupts even if they are unmaskable b) `val nmi` is part of the `TileInterrupts` bundle, not it's own bundle. But I also don't have a strong reason to say there is a problem with doing it this way either.